src/main/java/org/apache/commons/beanutils2/BeanUtilsBean.java

```diff
@@ -1067,4 +1039,60 @@ public void setProperty(final Object bean, String name, final Object value)
                 (e, "Cannot set " + propName);
         }
     }
+
+    /**
+     * <p>Build the stringBuilder by using set/copy bean property for log, only
```

Comment:
This comment is not quite right IMO: It mentions logging but the method does not log or check log levels. Since it is a protected method, it can be called from a subclass so you have no idea how it will be used. All of this to say, I think the Javadoc should just say what it does.